examples/redirect/src/tests.rs

```diff
@@ -0,0 +1,38 @@
+use super::rocket;
+use rocket::testing::MockRequest;
+use rocket::Response;
+use rocket::http::Method::*;
+use rocket::http::Status;
+
+macro_rules! run_test {
+    ($path:expr, $test_fn:expr) => ({
+        let rocket = rocket::ignite().mount("/", routes![super::root, super::login]);
+        let mut request = MockRequest::new(Get, format!($path));
+
+        $test_fn(request.dispatch_with(&rocket));
+    })
+}
+
+#[test]
+fn test_root() {
+    run_test!("/", |mut response: Response| {
+        assert!(response.body().is_none());
+        assert_eq!(response.status(), Status::SeeOther);
+        for h in response.headers() {
+            match h.name.as_ref() {
```

Comment:
Due to recent changes, this will have to be changed to `h.name.as_str()`. Sorry about that!